src/task11/task11.js

```diff
@@ -1,8 +1,10 @@
-var obj = JSON.parse(ANCESTRY_FILE);
+var ancestry = JSON.parse(ANCESTRY_FILE);
+
+const average = array => array.reduce((a, b) => a + b) / array.length;
 
 function groupBy(items, func) {
     let centuries = [];
-    items.foreach(function (item) {
+    items.map(function (item) {
```

Comment:
Use **reduce** here.

Reply:
You dont need **centuries** variable
Hint, what does foreach/map/filter/reduce return ?

Reply:
and =>

Reply:
for the life of me , I don't know how to get rid of centuries
your hint is useful for the other function and task 10, but here it's an array of objects, not of numbers. So I need to push these values somewhere